docs/src/modules/utils/parseMarkdown.js

```diff
@@ -130,19 +131,32 @@ export function prepareMarkdown(config) {
 
   const demos = {};
   const docs = {};
+  const headingHashes = {};
+
+  // Process the English markdown before the other locales.
+  let filenames = [];
   requireRaw.keys().forEach((filename) => {
+    if (filename.match(notEnglishMarkdownRegExp)) {
+      filenames.push(filename);
+    } else {
+      filenames = [filename].concat(filenames);
```

Comment:
If english comes after the non-english one for some reason this removes the previous locales, no?

Maybe I wasn't clear with my "partition" strategy: I'd sort by "english or not?" first and then by locale. That way you have a the markdown files as a list where the first entry is always the english locale and the tail is stable.